RNA-seq/00b-additional_approaches.md

```diff
@@ -45,33 +48,30 @@ Run the following command:
 ```bash
 multiqc \
   QC/gastric_cancer \
```

Comment:
```suggestion
  ~/shared-data/training-modules/RNA-seq/QC/gastric-cancer \
```

Reply:
Thought we might want to use their own directory here but I guess making the example parallel is less confusing.

Reply:
I think if we are trying to demonstrate the full power of this fully operational multiQC, we would include all the QC data. But I could be convinced that the smaller set is fine!

Reply:
I think what you suggested here is better.